lib/cct/commands/openstack.rb

```diff
@@ -69,15 +74,29 @@ def create name, options={}
           OpenStruct.new(shell_parse(exec!(all_params).output))
         end
 
+        def add name, options={}
+          params.clear
+          yield params
+          all_params = ["add", name, "--format=shell"].concat(params.extract!(options))
+          OpenStruct.new(shell_parse(exec!(all_params).output))
+        end
+
+        def set name, options={}
+          params.clear
+          yield params
+          all_params = ["set", name].concat(params.extract!(options))
+          OpenStruct.new(shell_parse(exec!(all_params).output))
+        end
+
         def delete id_or_name
           params.clear
           exec!("delete", id_or_name)
         end
 
-        def list
+        def list *options
           params.clear
-          user_row = Struct.new(:id, :name)
-          result = exec!("list", "--format=csv").output
+          user_row = Struct.new(:id, :name, :subnets)
```

Comment:
@vmoravec Maybe it would be better to rewrite this as a OpenStruct. I guess `network` won't be the only component that has a third (or fourth) column.